src/whosthatpokemon/main.py

```diff
@@ -76,7 +76,7 @@ def refresh_question():
     # create a window and background image
     window = Tk()
     style = Style()
-    style.configure('TButton', font=('helvetica', 14, 'bold'), foreground='#2a75bb', highlightbackground='#ffcb05', relief='RAISED')
+    style.configure('TButton', font=('helvetica', 14, 'bold'), foreground='#ffffff', highlightbackground='#ffcb05', relief='raised')
```

Comment:
White on white? Is this working on your laptop? The button background does not work on mac, it's a known bug I guess. There's a workaround though if we really want it not white, but I figured it's okay

Reply:
You can remove highlightbackground='#ffcb05' entirely to avoid confusion